dox/doxygen/doxyfile.in

```diff
@@ -956,12 +956,6 @@ EXAMPLE_PATTERNS       = *
 
 EXAMPLE_RECURSIVE      = NO
 
-# The IMAGE_PATH tag can be used to specify one or more files or directories
-# that contain images that are to be included in the documentation (see the
-# \image command).
-
-IMAGE_PATH             = "@ELASTIX_DOXYGEN_DIR@/images"
-
```

Comment:
i would rather leave this one empty over deleting it.

Reply:
OK!

Reply:
Adjusted with my latest amend + force-push!